packages/flutter/lib/src/material/text_form_field.dart

```diff
@@ -332,7 +332,9 @@ class TextFormField extends FormField<String> {
     BuildContext context,
     EditableTextState editableTextState,
   ) {
-    if (defaultTargetPlatform == TargetPlatform.iOS && SystemContextMenu.isSupported(context)) {
+    if (defaultTargetPlatform == TargetPlatform.iOS &&
```

Comment:
this should be changed to `SystemContextMenu.isSupportedByField(editableTextState)` I think.